custom/opm/reports.py

```diff
@@ -908,6 +908,9 @@ def rows(self):
             rows.sort(key=lambda x: x[col_id], reverse=True)
         self._store_rows_in_redis(rows)
 
+        for i, row in enumerate(rows):
+            row[-1] = i + 1
```

Comment:
for this it might be more future proof to use:
```python
row[self.column_index('serial_number')] = i + 1
```
in case we add another column at some point.